lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -47,13 +49,22 @@
 @cache_control(no_cache=True, no_store=True, must_revalidate=True)
 def instructor_dashboard_2(request, course_id):
     """ Display the instructor dashboard for a course. """
-    course_key = SlashSeparatedCourseKey.from_deprecated_string(course_id)
+    try:
+        course_key = CourseKey.from_string(course_id)
+    except InvalidKeyError:
+        log.error(
+            u"Unable to find course with course key {course_key} while loading the Instructor Dashboard."
+            .format(course_key=course_id)
+        )
+        return HttpResponseServerError()
+
     course = get_course_by_id(course_key, depth=None)
```

Comment:
yea, another one of these. That should be depth=0 or just remove the optional parameter which defaults to 0. I know you didn't touch that, but while you are in here, can you do that? Thanks.